llvm/test/tools/llvm-objdump/BPF/disassemble-symbolize-operands.s

```diff
@@ -0,0 +1,24 @@
+# REQUIRES: bpf-registered-target
+
+## Verify generation of 'Lxx' labels for local jump targets,
+## when --symbolize-operands option is specified.
+
+# RUN: llvm-mc -triple=bpfel %s -filetype=obj -o - | \
```

Comment:
We prefer that the object file is written to the disk (e.g. `%t`) to ease debugging.